lib/kitchen/note_element.rb

```diff
@@ -62,10 +58,26 @@ def self.is_the_element_class_for?(node)
 
     protected
 
-    def translation_key_in(possible_translation_keys)
-      keys = possible_translation_keys & classes
+    def key
+      if I18n.t(document.short_name).instance_of?(String)
+        puts "Entry for book with short name #{document.short_name} does not exist in locales"
+        return
+      end
+      unless I18n.t(document.short_name).key?(:notes)
+        puts "The book #{document.short_name} does not have an entry for notes in locales"
```

Comment:
Can you tell me which of the `puts` and `raise` things here should stay and which should become `RecipeError` or other?

Reply:
For developers, it seems to me more useful to have `puts`/ a non-fatal response, so that the recipe can still run even if the developer hasn't made a locales file yet. This `puts` feedback also supplies the list of keys that the recipe expects to find in locales, which may be helpful in building the file. But is there a better way of doing this with `raise`/`rescue`?